lms/djangoapps/teams/tests/test_views.py

```diff
@@ -509,7 +516,7 @@ def test_expand_public_user(self):
         ('queryable', ['Search']),
         ('Tonga', ['Search']),
         ('Island', ['Search']),
-        ('search queryable', []),
+        ('not-a-query', []),
         ('team', ['Another Team', 'Public Profile Team']),
```

Comment:
Also just to clarify - the line you commented on here isn't actually part of this PR, it just got caught up in the rebasing I was doing. Did you mean to add something to https://github.com/edx/edx-platform/pull/9613?